provider/data/datetime/fingerprints.csv

```diff
@@ -27500,204 +27500,266 @@ time_zone/essentials@1, ur-IN, -> und
 time_zone/essentials@1, vec, -> eo
 time_zone/essentials@1, xnr, 173B, 81B, 331c125b1ab74aab
 time_zone/essentials@1, yo, 109B, 17B, 6f563aca2d7a093
-time_zone/exemplars@1, <lookup>, 969B, 196 identifiers
-time_zone/exemplars@1, <total>, 488149B, 480228B, 179 unique payloads
+time_zone/exemplars@1, <lookup>, 885B, 182 identifiers
+time_zone/exemplars@1, <total>, 244737B, 238270B, 146 unique payloads
```

Comment:
This size went down, and then we have

```
time_zone/exemplars_root@1, <total>, 167494B, 166036B, 33 unique payloads
```

So there is an overall size decrease, yes?

Reply:
Yes, about 10%. Actually the size decrease seems to be better for core languages, as many moderate languages fall back to und values instead of something in their script, which increases their data size.